tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Maybe the model deserves a separate small section with 1/2 sentences above this one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I've added this information to the classification task section</p>